spark/src/main/scala/org/apache/spark/sql/delta/commands/DeleteCommand.scala

```diff
@@ -267,6 +272,8 @@ case class DeleteCommand(
                 touchedFiles,
                 txn.snapshot)
               metrics("numDeletedRows").set(metricMap("numDeletedRows"))
+              numDeletionVectorsAdded = metricMap("numDeletionVectorsAdded")
+              numDeletionVectorsRemoved = metricMap("numDeletionVectorsRemoved")
               numRemovedFiles = metricMap("numRemovedFiles")
               actions
             } else {
```

Comment:
what about the case where DV DELETE is not enabled? The underlying table could have DVs.

Reply:
+1 we should track removed DV there